### ISSUE TITLE
Issue with "ARIA: combobox role": Combobox should contain a text field and options as children

### ISSUE THREAD

**Issue by lopis** [mdn/content#14784]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/combobox_role

#### What information was incorrect, unhelpful, or incomplete?

In the example provided in this page, the combobox role is assigned to the `input` element. However, a combobox is a composite element that should _contain_ an input, as well as a list of options, as children. Using it directly on the `input` element is incorrect.

#### Specific section or headline?

[The example](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/combobox_role#examples).

#### What did you expect to see?

The line `<div class="combo-wrap">` should read `<div class="combo-wrap" role="combobox">`. This way the `ul` is its child. The role should be removed from the `input`.

#### Did you test this? If so, how?

I used Lighthouse to test this. Lighthouse will complain if this rule is not followed.

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/accessibility/aria/roles/combobox_role`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/combobox_role
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/accessibility/aria/roles/combobox_role/index.md
* Last commit: https://github.com/mdn/content/commit/e752a8fe731fbdb9061e39a6e2b797a7ccb7e2b5
* Document last modified: 2022-01-19T17:57:58.000Z

</details>

**Comment by estelle** [comment]
https://github.com/mdn/content/pull/14534

**Comment by estelle** [comment]
my understanding is tht the combobox is the input, it "aria-haspopup".
We've written new documentation which is in PR at the moment, but in that example the role is on the input.

**Comment by lopis** [comment]
My understanding is that `haspopup` also goes in the parent. I'll be happy to be proven wrong, but the documentation I find isn't clear and Lighthouse disagrees. So perhaps it's Lighthouse that needs to be more flexible with the "children" of the combobox not being literally DOM children on the combobox element.

**Comment by lopis** [comment]
Created a discussion on Lighthouse too.

https://github.com/GoogleChrome/lighthouse/discussions/13836
